README.rst

```diff
@@ -1809,11 +1809,11 @@ Choose from the list of available rules:
 
   Standardize spaces around ternary operator.
 
-* **ternary_to_null_coalescing** [@PHP70Migration, @PHP71Migration]
+* **ternary_to_null_coalescing** [@PHP70Migration, @PHP71Migration, @PHP73Migration]
 
   Use ``null`` coalescing operator ``??`` where possible. Requires PHP >= 7.0.
 
-* **trailing_comma_in_multiline_array** [@Symfony, @PhpCsFixer]
+* **trailing_comma_in_multiline_array** [@Symfony, @PhpCsFixer, @PHP73Migration]
```

Comment:
how is it related to 7.3 ?

Reply:
the `after_heredoc` option is desired while migrating to php 7.3. That is the reason, why I've added the fixer with that option to the ruleset.